src/services/account/useAvailablePlans.ts

```diff
@@ -0,0 +1,77 @@
+import { useQuery } from '@tanstack/react-query'
+import { z } from 'zod'
+
+import Api from 'shared/api'
+
+const PlansSchema = z
+  .object({
+    owner: z
+      .object({
+        availablePlans: z
+          .array(
+            z
+              .object({
+                baseUnitPrice: z.number(),
+                benefits: z.array(z.string()),
+                billingRate: z.string().nullable(),
+                marketingName: z.string(),
+                monthlyUploadLimit: z.number().nullable(),
+                value: z.string(),
+              })
+              .nullish()
+          )
+          .nullish(),
```

Comment:
What is the reason for switching from `nullable()` to `nullish()`?

Reply:
Copy paste from an early days hook, good reminder of why we use nullable over nullish 👍